__tests__/db.test.js

```diff
@@ -0,0 +1,31 @@
+const app = require("../app");
+const request = require('supertest');
+const db = require("../db/connection")
+const seed = require("../db/seeds/seed")
+const testData = require("../db/data/test-data/index")
+
+beforeAll(() => seed(testData));
+afterAll(()=> db.end())
+
+describe("GET /api/topics", () =>{
+    test("Nonsense requests return a 404", () =>{
+        return request(app)
+        .get('/api/blueberry')
+    })
+    test("GET request to topics returns a 200", () =>{
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+    })
+    test("Returns data with 'slug' and 'description' tables", () =>{
+        return request(app)
+        .get('/api/topics')
+        .then(({body})=>{
+            body.forEach((topic)=>{
```

Comment:
Before any iteration in a test, we should check the length of the array to make sure there is indeed something to iterate over

Reply:
Checking it is greater than 0 is a good start but this could easily be just one element. We should check it's the length we expect